### DISCUSSION TITLE
Beautiful. I love it !

### DISCUSSION THREAD

**Discussion by serge-hulne** [maragudk/gomponents#199] [Show and tell]
I have done similar things: https://github.com/serge-hulne/ginko , but gomponents is quite beautiful.

**Comment by markuswustenberg** [comment]
Thanks! Oh yeah, cool. 😊

**Comment by serge-hulne** [comment]
I also built something similar with Crystal. Here are 3 examples, for inspiration: 
https://github.com/serge-hulne/isomorphic-crystal-examples/tree/main/src/ex1
